ui/shared/ad/CustomAdBanner.tsx

```diff
@@ -0,0 +1,48 @@
+import { Flex, chakra, Tooltip, Image } from '@chakra-ui/react';
+import React, { useState, useEffect } from 'react';
+
+import useCustomBanners from 'ui/shared/ad/useCustomBanners';
+
+const CustomAdBanner = ({ className }: { className?: string }) => {
+  const customBanners = useCustomBanners().data || [];
+  const [ currentBannerIndex, setCurrentBannerIndex ] = useState(0);
+  useEffect(() => {
+    if (customBanners.length === 0) {
+      return;
+    }
+    const timer = setInterval(() => {
+      setCurrentBannerIndex((prevIndex) => (prevIndex + 1) % customBanners.length);
+    }, 60000);
```

Comment:
I would suggest putting it in the config as well, so other users can change this interval.

Reply:
Ok, let me change this.